lib/main.coffee

```diff
@@ -38,6 +42,12 @@ module.exports =
       lintOnFly: true
       lint: (textEditor) =>
         filePath = textEditor.getPath()
+
+        onlyConfig = atom.config.get('linter-jshint.disableWhenNoJshintrcFileInPath')
+        jshintConfig = helpers.findFile(filePath, '.jshintrc')
```

Comment:
can you check for config file only when the setting is enabled please, 'cause we don't wanna find it when the config is disabled